src/seo/getAlternateLanguageLinks.ts

```diff
@@ -0,0 +1,78 @@
+import type { NewsroomLanguageSettings } from '@prezly/sdk';
+
+import type { AlternateLanguageLink } from '../components-nextjs/PageSeo';
+import type { LocaleObject } from '../intl';
+import { getFallbackLocale } from '../intl';
+
+import { createAlternateLanguageLink } from './utils/createAlternateLanguageLink';
+import { getDefaultHrefLang } from './utils/getDefaultHrefLang';
+import { mapLanguagesToLocales } from './utils/mapLanguagesToLocales';
+
+export function getAlternateLanguageLinks(
+    languages: NewsroomLanguageSettings[],
+    getTranslationUrl: (locale: LocaleObject) => string | undefined,
+    createHref: (locale: LocaleObject, translationUrl: string) => string,
+): AlternateLanguageLink[] {
+    const alternateLanguageFallbackGlobalLinks: AlternateLanguageLink[] = [];
+    const alternateLanguageLocales: LocaleObject[] = [];
+    const [localesByLangCode, defaultLangLocale] = mapLanguagesToLocales(languages);
+
+    localesByLangCode.forEach((locales, globalLocaleCode) => {
+        let hasGlobalLocale = false;
+
+        locales.forEach((locale) => {
+            alternateLanguageLocales.push(locale);
+
+            if (!hasGlobalLocale && locale.isGlobal) {
+                hasGlobalLocale = true;
+            }
+        });
```

Comment:
This can be simplified quite a bit.
```suggestion
        const hasGlobalLocale = locales.some((locale) => locale.isGlobal);
        alternateLanguageLocales.push(...locales.values());
```

Reply:
Also I tried to avoid additional array looping, I assume that `locales.values()` is mapping values to array

Reply:
Same as in my previous comment, the size of the array makes gains from such optimizations negligible.

Reply:
Fixed https://github.com/prezly/theme-kit-nextjs/pull/215/commits/b3d696fa8d957067f955b6c7be2f018206185bd5